docs/reference/glossary.asciidoc

```diff
@@ -134,11 +171,13 @@ then it will be auto-generated. (also see <<glossary-routing,routing>>)
 +
 --
 // tag::index-def[]
-An index is like a _table_ in a relational database. It has a
-<<glossary-mapping,mapping>> which contains a <<glossary-type,type>>,
-which contains the <<glossary-field,field>> in the index.
+// tag::index-def-short[]
+An optimized collection of documents where each document is a collection of fields, 
+which are the key-value pairs that contain your data. 
+// end::index-def-short[]
 
-An index is a logical namespace which maps to one or more
+An index is like a _table_ in a relational database. 
+It is a logical namespace that maps to one or more
```

Comment:
I feel like comparing an index to a table likely causes more harm than good.
```suggestion
An index is a logical namespace that maps to one or more
```

Reply:
Agreed...at one level, it's a useful analogy, but it has also proved to be a problematic one.